client/cli/lib/utils.ts

```diff
@@ -32,3 +32,26 @@ export const validateDependencies = (host: Dependencies, client: Dependencies) =
     }
   });
 };
+
+export const getComponentDirectories = (dir?: string, workspaces?: string[]) => {
+  let globs: string[];
+  if (dir) {
+    globs = [dir];
+  } else if (workspaces && workspaces.length > 0) {
+    globs = workspaces;
```

Comment:
Can we throw an error if both `dir` and `workspaces` are defined? not doing so can lead to a user expecting that we'll treat both workspaces and dir and not understanding why we only handle the dir...

Reply:
Dir is an override, if you only want to work on a single dir. You can't set it in the config file while you can set workspaces

Reply:
So our defined default is pwd?

Reply:
yes